source/beamline/SGM/energy/SGMEnergyCoordinatorControl.h

```diff
@@ -249,6 +251,11 @@ protected slots:
 	 */
 	virtual AMAction3* createMoveAction(SGMEnergyTrajectory* energyTrajectory);
 
+	/*!
+	  * Creates an action which returns the Energy Coordinator Control back to its defaults.
+	  */
+	virtual AMAction3* setDefaultsAction();
```

Comment:
Purely a pedantic thing, can you make this `createDefaultsAction()` or something similar?  It would match most of our other action getters code.